apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.php

```diff
@@ -7,6 +7,23 @@
 
 namespace A8C\FSE\ErrorReporting;
 
+/**
+ * Whether or not the site is eligible for Error Reporting, which is a feature that's
+ * specific to WPCOM.
+ *
+ * By default, sites should not be eligible.
+ *
+ * @return bool True if current site is eligible for error reporting, false otherwise.
+ */
+function is_site_eligible_for_error_reporting() {
+	/**
+	 * Can be used to toggle the Error Reporting functionality.
+	 *
+	 * @param bool true if Error Reporting should be enabled, false otherwise.
+	 */
+	return apply_filters( 'a8c_enable_error_reporting', false );
```

Comment:
Where are we defining this `a8c_enable_error_reporting`? I did not find any occurrences of it inside WPCOM 🤔